### ISSUE TITLE
SC2013 IFS setting igoring

### ISSUE THREAD

**Issue by PHPoenX** [koalaman/shellcheck#1708]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2013
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
IFS=$'\n'
for line in $(cat abcd)
do
echo "$line"
done

```

#### Here's what shellcheck currently says:
^-- SC2013: To read lines rather than words, pipe/redirect to a 'while read' loop.

#### Here's what I wanted or expected to see:
No issues detected(with current setting of IFS, bash will read lines)
